pkg/annotations/annotations.go

```diff
@@ -322,5 +322,6 @@ var AnnotationExpansions = map[string][]string{
 		WCOWDisableGMSA,
 		DisableWritableFileShares,
 		VSMBNoDirectMap,
+		HostProcessContainer,
```

Comment:
shouldn't this be set to `false`, when disable unsafe annotations is `true`?

Reply:
oh yeah, i missed that
either an annotation for `DisableHostProcessContainer` needs to be created, or we need an `AnnotationDenyList` thingy

Reply:
@helsaawy I added a `DisableHostProcessContainer` annotation and updated the PR